src/App.tsx

```diff
@@ -15,36 +17,103 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortField {
+  name = 'name',
+  length = 'length',
+}
+
+interface Filter {
+  field: SortField;
+  isreverse: boolean;
+}
+
+const preparedGoods = (
+  goods: string[],
+  { field, isreverse }: Filter,
+): string[] => {
+  const sortedGoods = [...goods];
+
+  if (!field) {
```

Comment:
The check `if (!field)` is unnecessary because `field` should always be provided as part of the `Filter` interface. Consider removing this condition and ensuring `field` is always set to a valid `SortField` value.